vector-stores/spring-ai-gemfire/src/main/java/org/springframework/ai/vectorstore/GemFireVectorStore.java

```diff
@@ -70,20 +74,22 @@ public static final class GemFireVectorStoreConfig {
 
 		private final int topKPerBucket;
 
+		public final int topK;
+
 		private final String documentField;
 
 		public static Builder builder() {
 			return new Builder();
 		}
 
 		private GemFireVectorStoreConfig(Builder builder) {
-			String base = UriComponentsBuilder
-				.fromUriString("http{ssl}://{host}:{port}/gemfire-vectordb/v1/indexes/{index}")
+			String base = UriComponentsBuilder.fromUriString(DEFAULT_URI)
```

Comment:
The default uri isn't a 1:1 replacement.  Is that intentional?
It is now:         http{ssl}://{host}:{port}/gemfire-vectordb/v1/indexes
It used to be: http{ssl}://{host}:{port}/gemfire-vectordb/v1/indexes/{index}

Will the builder.index be used in this case?  it looks like it would have been the replacement for {index} but that no longer exists?

Reply:
corrected it